docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -226,7 +228,19 @@ of these operations.
 
 [discrete]
 [[agent-policy-scaling-recommendations]]
-== Policy scaling recommendations
+== Scaling recommendations
+
+**{agent} policies**
 
 A single instance of {fleet} supports a maximum of 500 {agent} policies. If more policies are configured, UI performance might be impacted.
 
+**{agents}**
+
+When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `max number of agents * 2`.
+
+For example, with 25,000 running {agents} you could set the cache value to `50,000`:
```

Comment:
if you're using code formatting for the #, the formatting should match the yaml below. could also remove the code formatting

```suggestion
For example, with 25,000 running {agents} you could set the cache value to `50000`:
```